include/gmsh.h

```diff
@@ -25,20 +25,28 @@ class GMSH : public Mesh {
   void read_elements(const std::string& filename);
 
   //! Text output
```

Comment:
This was there, and I haven't changed it yet. Let me, the next time I come across anything, I will fix it